### PR TITLE
[FIX] Include acc templates from parent CoA templates

### PR DESCRIPTION
**Pull request by pablocm-aserti**
Despite account_chart_update using the same [search criteria](https://github.com/OCA/account-financial-tools/blob/8.0/account_chart_update/wizard/wizard_chart_update.py#L707-L714) than the [CoA installer](https://github.com/odoo/odoo/blob/05179389bf78badad26ca5198e81f0f8c708eb53/addons/account/account.py#L2599-L2602) to look for account templates to compare with the installed CoA, it was missing accounts defined in parent CoA templates.
It was caused because because the installer [recursively installs](https://github.com/odoo/odoo/blob/05179389bf78badad26ca5198e81f0f8c708eb53/addons/account/account.py#L3306-L3307) parent CoAs, while account_chart_update doesn't. As a result, changes made to acc templates defined in parent CoA templates were not beign detected, and those accounts weren't beign updated.

### CODE REVIEW

**Comment by pablocm-aserti** [comment]
Fixes https://github.com/OCA/l10n-spain/issues/307
Ping @pedrobaeza

File: account_chart_update/wizard/wizard_chart_update.py [hunk 1]
```diff
@@ -712,6 +706,21 @@ def _find_accounts(self, mapping_accounts):
             acc_templ_criteria += [
                 '&', ('parent_id', 'child_of', [root_account_id]),
                 ('chart_template_id', '=', False)]
+        if chart_template.parent_id:
```
Comment (pedrobaeza) [inline account_chart_update/wizard/wizard_chart_update.py:709]: This should be recursive: you can have more a grand-parent, grand-grand-parent, and so.

Comment (pablocm-aserti) [inline account_chart_update/wizard/wizard_chart_update.py:709]: It is, just 2 lines below (711)

Comment (pedrobaeza) [inline account_chart_update/wizard/wizard_chart_update.py:709]: Sorry, I didn't see it! Sunday afternoon...

**Comment by pedrobaeza** [comment]
:+1: 

cc @antespi @rafaelbn

**Comment by rafaelbn** [comment]
Travis fails

**Comment by pedrobaeza** [comment]
The Travis error is tracked in https://github.com/OCA/account-financial-tools/issues/352, but unrelated to this.

**Comment by rafaelbn** [comment]
thanks! :+1:

**Comment by pedrobaeza** [comment]
@sbidoul, your review is also appreciated here.

**Pull request merged**